README.md

```diff
@@ -1,2 +1,222 @@
 # stellar-dbt-public
 Public DBT instance to aid in data transformation for analytics purposes
+
+## Table of Contents
+- [Data Transformation](#data-transformation)
+    - [Workflow](#workflow)
+    - [dbt Project Structure](#dbt-project-structure)
+    - [Tests](#tests)
+- [Getting Started](#getting-started)
+    - [Oauth connection](#oauth)
+- [dbt Setup](#dbt-setup)    
+- [Working with dbt](#working-with-dbt)
+    - [Running Models](#running-models)
+    - [Running Tests](#running-tests)
+- [Project Structure](#project-structure)
+    - [Development Folders](#development-folders)
+
+
+## Data Transformation 
```

Comment:
This might be better named as `dbt Overview` but I don't feel super strongly about it